### PR TITLE
[for-developers] update docs to divio

### PR DESCRIPTION
**Pull request by mustafaStakater**

### CODE REVIEW

File: content/for-developers/how-to-guides/build-and-push-your-image/build-and-push-your-image.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Lets push the image to nexus docker repo.
+```sh
+# Buildah push Info https://manpages.ubuntu.com/manpages/impish/man1/buildah-push.1.html
+buildah push <nexus-docker-reg-url>/<app-name>:1.0.0 docker://<nexus-docker-reg-url>/<tenant-name>/<app-name>:1.0.0
+```
+
+## Verify Image Available
+
+Login to Nexus and Verify that the image is available on the cluster.
```
Comment (rasheedamir) [inline content/for-developers/how-to-guides/build-and-push-your-image/build-and-push-your-image.md:58]: @mustafaStakater lets add screenshot to show where to find the image in nexus

File: content/for-developers/how-to-guides/package-and-push-your-chart/package-and-push-your-chart.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+   This command packages a chart into a versioned chart archive file.
+
+1. Upload packaged chart to Nexus Helm Registry.
+
+   ```sh
+   curl -u "<helm_user>":"<helm_password>" NEXUS_HELM_REG_URL --upload-file "CHART_NAME-CHART_VERSION.tgz"
+   ```
+
+   > Make sure to get credentials from Stakater Admin.
+
```
Comment (rasheedamir) [inline content/for-developers/how-to-guides/package-and-push-your-chart/package-and-push-your-chart.md:31]: @mustafaStakater lets add screenshot to show where to find the chart in nexus

File: content/for-developers/tutorials/outer-loop/01-access-cluster/access-cluster.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
 # Access Your Cluster
```
Comment (rasheedamir) [inline content/for-developers/tutorials/outer-loop/01-access-cluster/access-cluster.md:1]: @mustafaStakater this is required in inner loop as well so, moved to outer loop section?

Comment (mustafaStakater) [reply content/for-developers/tutorials/outer-loop/01-access-cluster/access-cluster.md:1]: should we keep it in both section or move it to how-to section

Comment (mustafaStakater) [reply content/for-developers/tutorials/outer-loop/01-access-cluster/access-cluster.md:1]: also this document logs the user in with saap frontend, we will remove it for inner loop docs

**Review by AsfaMumtaz — APPROVED** [review]

**Pull request merged**